xbmc/pictures/metadata/ImageMetadataParser.cpp

```diff
@@ -16,6 +16,14 @@
 #include <limits>
 #include <vector>
 
+#if (EXIV2_MAJOR_VERSION == 0) && (EXIV2_MINOR_VERSION < 28)
+#define EXIV_UINT32 toLong()
```

Comment:
I would remove the braces from the defines, so the actual function calls look like function calls. Currently those look like accessing member variables.
Maybe also name the defines `EXIV_toUint32` and `EXIV_toInt64` to be more consistent with the current API names.